processor/src/main/java/tw/ktrssreader/processor/util/TypeExtensions.kt

```diff
@@ -113,33 +100,16 @@ fun String.getVariableName(tag: String): String {
 
 fun String.appendTypeConversion(typeString: String): String {
     return when {
-        typeString.contains(String::class.java.simpleName, ignoreCase = true) -> this
-        typeString.contains(Integer::class.java.simpleName, ignoreCase = true) -> "$this?.toIntOrNull()"
-        typeString.contains(Boolean::class.java.simpleName, ignoreCase = true) -> "$this?.%M()"
-        typeString.contains(Long::class.java.simpleName, ignoreCase = true) -> "$this?.toLongOrNull()"
-        typeString.contains(Short::class.java.simpleName, ignoreCase = true) -> "$this?.toShortOrNull()"
+        typeString.contains(String::class.simpleName!!, ignoreCase = true) -> this
+        typeString.contains(Integer::class.simpleName!!, ignoreCase = true) -> "$this?.toIntOrNull()"
```

Comment:
Replace with `Int`?

Reply:
Yea, I use the wrong type. Thanks for the heads up! I modified it, please check it out.